product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -67,28 +73,40 @@ Once your identity provider is set up, you can view your connection status, ID,
 !!!SeeAlso "Further reading"
    [Setting up specific identity providers](knowledge_articles)
 
-## Manage roles for added users
+### Add a domain
+
+A verified domain is necessary for your users to have a streamlined login experience. 
+
+In the **Domains** tab, enter the domain and select **Next: Verify Domain**. Copy the TXT record and follow the instructions. Select **Done**. 
+
+Your domain and its status appears on the **Domains** tab, where you can delete or verify it. To add another domain, select **Add Domain**. 
+
+When you have at least one verified domain, the identity provider status becomes **Active** on the **Identity Providers** tab. The status becomes **Inactive** when the domain is no longer verified. 
+
+!!! Note
+    The identity provider status can take up to three minutes to update.
+
+### Manage roles for added users
 
 You add users through your identity provider. A user who you add in the identity provider is automatically added to BigAnimal. BigAnimal assigns them with the default role of organization member. You manage roles and permissions from BigAnimal. See [Managing portal access](/biganimal/latest/administering_cluster/01_portal_access/).
 
 !!! Note
 
     A user is created in BigAnimal only after they log in. After they log in, you can change their BigAnimal role.
 
-
-## Add a tile
+### Add a tile
 
 Once you establish the identity provider, you can create a BigAnimal tile for users to access the organization's BigAnimal application. To do so, copy the quick sign-in URL from the **Settings > Identity Provider** page of the BigAnimal portal. For details on how to add a tile, refer to your identify provider documentation for instructions on setting up SSO access to your application.
 
 
 ## Next steps
 
-You and other users can log in to BigAnimal using your identity provider credentials.
+You and other users can log in to BigAnimal using your identity provider credentials. 
 
 You can rename the default project BigAnimal creates for you. See [Editing a project](/biganimal/release/administering_cluster/projects/#editing-a-project).
 
-You can [set up your cloud service provider](/biganimal/latest/getting_started/02_connecting_to_your_cloud/) so that you or other users with the correct permissions can create clusters.
+If using your own cloud account, you can [set up your cloud service provider](/biganimal/latest/getting_started/02_connecting_to_your_cloud/) so that you or other users with the correct permissions can create clusters.
 
-After setting up your cloud server provider, you can assign roles to your default project. See [Managing portal access](/biganimal/release/administering_cluster/01_portal_access).
+- You can assign roles to your default project. See [Managing portal access](/biganimal/release/administering_cluster/01_portal_access). (If using your own cloud account, you need to set it up first.) 
```

Comment:
> (If using your own cloud account, you need to set it up first.) 

this line is confusing in this context as `cloud account != identity provider` so here we may mean the `If using your own identity provider, you need to set it up first.`